docs/content/preview/features.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Benefits of using YugabyteDB
+headerTitle: Key benefits
+linkTitle: Key benefits
+description: Benefits of using YugabyteDB to power your next cloud-native application.
+headcontent: Why YugabyteDB should power your next cloud-native application
+type: docs
+---
+
+YugabyteDB is a high-performance distributed SQL database for powering global, internet-scale applications. Built using a combination of high-performance document store, per-shard distributed consensus replication, and multi-shard ACID transactions (inspired by Google Spanner), YugabyteDB serves both scale-out RDBMS and internet-scale OLTP workloads with low query latency, extreme resilience against failures, and global data distribution. As a cloud-native database, it can be deployed across public and private clouds as well as in Kubernetes environments.
```

Comment:
Could we add this on the "Documentation Home" page too, just before the "Get started in the Cloud" box ?